### ISSUE TITLE
AttributeError: 'PosixPath' object has no attribute 'format'

### ISSUE THREAD

**Issue by lujanrr** [pyannote/pyannote-database#74]
Im trying to train the model with my own data, but i cannot solve this problem.

**To Reproduce**

`$pyannote-audio sad train --subset=train --to=200 --parallel=4 ${EXP_DIR} Test.SpeakerDiarization.OwnData
`

My database.yml :
```
*Databases:
   Test: ./data_set_wav/{uri}.wav
   MUSAN: ./Pyannote/AMI/musan/{uri}.wav

Protocols:
   Test:
      SpeakerDiarization:
         OwnData:
           train:
              uri: ./Reference_files/validate/train.data.lst
              annotation: ./Reference_files/validate/train.data.rttm
              annotated: ./Reference_files/validate/train.data.uem
           development:
              uri: ./Pyannote/AMI/AMI/MixHeadset.development.lst
              annotation: ./Pyannote/AMI/AMI/MixHeadset.development.rttm
              annotated: ./Pyannote/AMI/AMI/MixHeadset.development.uem
           test:
              uri: ./Pyannote/AMI/AMI/MixHeadset.test.lst
              annotation: ./Pyannote/AMI/AMI/MixHeadset.test.rttm
              annotated: ./Pyannote/AMI/AMI/MixHeadset.test.uem
   MUSAN:
      Collection:
         BackgroundNoise:
            uri: ./Pyannote/AMI/musan/MUSAN/background_noise.txt
         Noise:
            uri: ./Pyannote/AMI/musan/MUSAN/noise.txt
         Music:
            uri: ./Pyannote/AMI/musan/MUSAN/music.txt
         Speech:
            uri: ./Pyannote/AMI/musan/MUSAN/speech.txt

```
**pyannote environment**

```
pyannote.audio==1.1.1
pyannote.core==4.3
pyannote.database==4.1.1
pyannote.metrics==3.1
pyannote.pipeline==1.5.2
```
 My  train.lst file (First 10 lines):
```
140471632__701151021998050417_272_2751_20210210_134647
127467523__701151985360118_272_2754_20210201_095339
135434197__701151031971345585_356_3972_20210205_165315
131519247__701151957207998_461_2881_20210203_152337
147034889__450198019993417681_93_3989_20210216_094739
130398174__701151091989826532_272_2754_20210203_084818
128654151__701151019988926593_356_3956_20210201_181652
138260478__01577999895159_147_2413_20210209_111043
146777865__701151021996438304_272_4027_20210215_183913
81808790__450198986402390_137_2680_20201217_085743
```

My train.rttm file (First 10 lines):
```
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 174.301 1.25 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 56.4404 0.9400000000000048 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 20.6501 1.2001000000000026 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 23.8802 1.110000000000003 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 28.2202 0.6000000000000014 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 3.63 0.9199999999999999 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 5.63 0.41000000000000014 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 7.4301 0.8999999999999995 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 8.4901 0.6199999999999992 <NA> <NA> Customer <NA> <NA>
SPEAKER 80596681__701151988339062_105_2448_20201216_114632 1 9.7601 0.7100000000000009 <NA> <NA> Customer <NA> <NA>
```
**Additional context**
The example with the amicorpus data run with no error, but when i try to train my own data always get this error

**Comment by Yuee98** [comment]
I encountered same problem. The reason for me is a mismatch among uem, rttm and lst. After checking and regenating  those files the problem got solved.

**Comment by hbredin** [comment]
Fixed by #78.
